common/cmake_modules/GncAddTest.cmake

```diff
@@ -117,11 +117,14 @@ function(gnc_gtest_configure)
   endif()
   unset(GTEST_SRC_DIR CACHE)
   if (GTEST_ROOT)
-    find_path(GTEST_INCLUDE_DIR gtest/gtest.h
-      HINTS ${GTEST_ROOT}/include ${GMOCK_ROOT}/gtest/include)
     find_path(GTEST_SRC_DIR src/gtest-all.cc
       HINTS ${GTEST_ROOT} ${GMOCK_ROOT}/gtest /usr/src/gtest)
-    if (GTEST_SRC_DIR AND GTEST_INCLUDE_DIR)
+    if (GTEST_SRC_DIR)
+      if (EXISTS ${GTEST_SRC_DIR}/include/gtest/gtest.h)
+        set(GTEST_INCLUDE_DIR ${GTEST_SRC_DIR}/include CACHE PATH "" FORCE)
+      else()
+        message(FATAL_ERROR "GTEST sources found, but no include directory")
```

Comment:
Say "doesn't have 'gtest/gtest.h'". It might well have an include directory and telling the user that it doesn't would be confusing and slow down troubleshooting.

Reply:
What do you recommend?